src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -95,7 +95,15 @@ if [ "$(uname -s)" == "Darwin" ]; then
 elif [ "$(uname -s)" == "Linux" ]; then
   # On Linux, we'll enable core file generation unconditionally, and if a dump
   # is generated, we will print some useful information from it and delete the
-  # dump immediately. 
+  # dump immediately.
+
+  if [ -e /proc/self/coredump_filter ]; then
```

Comment:
> I like the idea of capturing dumps but is there a better place then the runner scripts to do it?

The other concern that lead us to this solution was the desire to minimize the number of dumps captured in cases where the entire run-time had problems.  There was fear about filling up the machines with dumps, so the tests happen right before we run tests based on if any other dump is present.

Long term, we would like dumps to be collected anywhere we run tests, I would think.  Do you have a thought about how we would centralize this infrastructure across Jenkins, Helix and local dev boxes?